docs/products/kafka/kafka-connect/howto/s3-sink-connector-aiven.rst

```diff
@@ -0,0 +1,115 @@
+
+* ``AWS_S3_NAME``: The name of the S3 bucket
+* ``AWS_S3_REGION``: The AWS region where the S3 bucket has been created
+* ``AWS_USER_ACCESS_KEY_ID``: The AWS user access key ID
+* ``AWS_USER_SECRET_ACCESS_KEY``: The AWS user secret access key
+
+Setup an S3 sink connector with Aiven CLI
+-------------------------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka Connect® S3 sink connector using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect® configuration file
+''''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``s3_sink.json``) with the following content:
+
+::
+
+    {
+        "name": "<CONECTOR_NAME>",
+        "connector.class": "io.aiven.kafka.connect.s3.AivenKafkaConnectS3SinkConnector",
+        "key.converter": "org.apache.kafka.connect.converters.ByteArrayConverter"",
+        "value.converter": "org.apache.kafka.connect.converters.ByteArrayConverter",
+        "topics": "<TOPIC_NAME>",
+        "aws.access.key.id": "<AWS_USER_ACCESS_KEY_ID>",
+        "aws.secret.access.key": "<AWS_USER_SECRET_ACCESS_KEY>",
+        "aws.s3.bucket.name": "<AWS_S3_NAME>",
+        "aws.s3.region": "<AWS_S3_REGION>"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``topics``: the list of Apache Kafka® topics to sink to the S3 bucket
+* ``key.converter`` and ``value.converter``: data converters, depend on the topic data format, check the `GitHub repository documentation <https://github.com/aiven/s3-connector-for-apache-kafka>`_ for more information
+* ``aws.access.key.id``: the AWS user access key ID
+* ``aws.secret.access.key``: The AWS user secret access key
+* ``aws.s3.bucket.name``: The name of the S3 bucket
+* ``aws.s3.region``: The AWS region where the S3 bucket has been created
+
+.. Tip::
+
+    You can define S3 sink connector naming and data formats by setting the :doc:`dedicated parameters <../concepts/s3-sink-additional-parameters>`.
+
+
+
+Check out the `GitHub repository parameters documentation <https://github.com/aiven/aiven-kafka-connect-s3>`_ for the full list of configuration options.
+
+
+Create a S3 sink connector with Aiven CLI
+'''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, execute the following :ref:`Aiven CLI command <avn_service_connector_create>`, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
+
+:: 
+
+    avn service connector create SERVICE_NAME @s3_sink.json
+
+Check the connector status with the following command, replacing the ``SERVICE_NAME`` with the Aiven service and the ``CONNECTOR_NAME`` with the name of the connector defined before:
+
+::
+
+    avn service connector status SERVICE_NAME CONNECTOR_NAME
+
+Verify in the target S3 bucket, the presence of the topic and the data
```

Comment:
Should we add for a bit more clarity - "With the connection in place, verify that the data is flowing to the target S3 bucket" ? At the moment the sentence seem to be abrupt.